server/views/formPages/incidentMacros.njk

```diff
@@ -184,53 +185,67 @@
 
 {% endmacro %}
 
+<!-- this one -->
 {% macro radiosWithNestedTextbox(question) %}
 
-<div class="govuk-form-group">
+{% if question.primaryQuestion.errorMessage.text %}
+{% set govukFormGroupErrorOuter  =  'govuk-form-group--error' %}
+{% set primaryErrorMessageText = question.primaryQuestion.errorMessage.text %}
+{% endif %}
+
+{% if not question.primaryQuestion.errorMessage and question.followUpQuestion.errorMessage.text %}
+{% set govukFormGroupErrorInner  =  'govuk-form-group--error' %}
+{% set followUpErrorMessageText = question.followUpQuestion.errorMessage.text %}
+{% endif %}
+
+<div class="govuk-form-group {{govukFormGroupErrorOuter}}">
   <fieldset class="govuk-fieldset">
     <!-- primary -->
     <legend class="govuk-fieldset__legend govuk-fieldset__legend">
       {{question.primaryQuestion.text}}
     </legend>
+    <span class="govuk-error-message">
+      <span class="govuk-visually-hidden">Error:</span>
```

Comment:
isn't this always going to be present for screen readers?

Reply:
I didn't think it would be. I took it out of https://design-system.service.gov.uk/components/error-message/
and if you look at another example that doesn't have errors like https://design-system.service.gov.uk/components/fieldset/
, the govuk-visually-hidden is included there too. 
So my thoughts were that it will not be seen by screen readers unless it's active ?